test/util.js

```diff
@@ -1,18 +0,0 @@
-import assert from 'assert';
-import { objectsEqual } from '../src/util.js';
```

Comment:
see above

Reply:
we could ad it back, but it feels weird to add dead code :wink:

Reply:
ok, let's get rid of it for now...